frontend/tests/controllers/UserPrivilegesTest.php

```diff
@@ -48,6 +48,77 @@ public function testAddRemoveRoles() {
         $this->assertNotContains('Mentor', $systemRoles);
     }
 
+    public function testPreviouslyAddedRoles() {
+        $username = 'testuserrole';
+        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser(
+            new \OmegaUp\Test\Factories\UserParams(
+                ['username' => $username]
+            )
+        );
+
+        $login = self::login($identity);
+        // Call to API Add Role
+        $response = \OmegaUp\Controllers\User::apiAddRole(
+            new \OmegaUp\Request([
+                'auth_token' => $login->auth_token,
+                'username' => $username,
+                'role' => 'Admin'
+            ])
+        );
+        $response = \OmegaUp\Controllers\User::apiAddRole(
```

Comment:
```suggestion
        \OmegaUp\Controllers\User::apiAddRole(
            new \OmegaUp\Request([
                'auth_token' => $login->auth_token,
                'username' => $username,
                'role' => 'Admin'
            ])
        );
        \OmegaUp\Controllers\User::apiAddRole(
```

si algo no se usa, no hay necesidad de bautizarlo.

Reply:
Listo.